assets/scss/nhsl-blocks.scss

```diff
@@ -97,3 +97,110 @@
   color: $color_nhsuk-white;
   width: 100%;
 }
+
+.nhsuk-promo .nhsuk-promo__content{
+  .nhsuk-image{
+    width: 100%;
+  }
+}
+
+// Image Block alternate style
+
+.is-style-nhsuk-image{
+  @extend .nhsuk-image;
+
+  figcaption{
+    margin: 0;
+  }
+}
+
+figure.is-style-nhsuk-table-responsive{
+  
+  @extend .nhsuk-table-responsive;
+
+  thead, tr, th, td {
+    border: 0;
```

Comment:
can we do something different for the admin view. It is nearly impossible to find a second or subsequent empty column to start typing into. UX is sub optimal. Looks beautiful on user side, but editing experience not ideal. i.e. can you tell this table has two columns to be able to use?
![Screenshot 2020-02-28 at 18 55 39](https://user-images.githubusercontent.com/44897304/75578592-03796580-5a5c-11ea-9357-ea49d9f85956.png)

Well, it actually has three!
![Screenshot 2020-02-28 at 18 55 48](https://user-images.githubusercontent.com/44897304/75578621-0d9b6400-5a5c-11ea-815c-28bef0471176.png)